### ISSUE TITLE
Arrays and objects are not expanded (or expandable)

### ISSUE THREAD

**Issue by juniorclearmetal** [lukehaas/RunJS#127]
In some cases where an array or object is a property of a parent object(s) it is represented as `Array` or `Object`, with no way I can find to expand them. `console.dir` does not work and there is no setting in preferences to do this. I can use `JSON.stringify` but that output is not formatted and hard to read. This limits the usefulness of RunJS because seeing the full output is not only helpful but often how I validate the functionality I am writing.

Examples:

```
const things = [
  {
    val: [
      {
        key: "1234",
        data: { location: "US" },
        label: "SAVANNAH, GEORGIA",
        value: "US"
      }
    ],
  }
];

const fnObj = () => ({
  item: things.map((thing, i) => ({thing}))
});

const fnArr = () => ({
  item: things.map((thing, i) => thing)
});

fnObj();
//=> { item: [ { thing: [Object] } ] }

fnArr();
//=> { item: [ { val: [Array] } ] }
```

The same thing in DevTools console:

<img width="281" alt="Screen Shot 2020-03-16 at 9 11 48 AM" src="https://user-images.githubusercontent.com/55420260/76777937-3747d100-6766-11ea-82d6-5e828e9b7ae7.png">

<img width="279" alt="Screen Shot 2020-03-16 at 9 11 01 AM" src="https://user-images.githubusercontent.com/55420260/76777950-3ca51b80-6766-11ea-9dc4-0a38e353e950.png">

**Comment by lukehaas** [comment]
@juniorclearmetal thanks for raising this. You're right, this isn't good. I will investigate and try and find a solution.

**Comment by lukehaas** [comment]
@juniorclearmetal this is now fixed in RunJS version 1.10.0.
You can now explore with infinite depth.

![Screenshot 2020-07-17 at 17 17 30](https://user-images.githubusercontent.com/741344/87808383-848d9980-c851-11ea-809e-ae2b8b18aa48.png)

Thanks again for raising this.
